database/migrations/2018_09_28_140458_add_user_type_to_voucher_states.php

```diff
@@ -47,12 +46,7 @@ public function up()
             UPDATE voucher_states SET user_type = 'CentreUser' 
             WHERE transition
             IN (
-                'bundle',
-                'disburse',
-                'unbundle-to-printed',
-                'unbundle-to-dispatched',
-                'lose',
-                'allocate'
+                'lose'
```

Comment:
Is this right - just 'lose' vouchers need the CentreUser type now?

Where has this migration been run? Would it be safer to write a new one to update previous state?

Reply:
It's only been run on staging. I'm prepared to scrub staging's db and reseed if this throws a clanger!